File: src/main/java/com/march/wxcube/module/ModalModule.java
package com.march.wxcube.module;

import com.march.common.utils.ToastUtils;
import com.taobao.weex.annotation.JSMethod;

/**
 * CreateAt : 2018/4/17
 * Describe : 模态
 *
 * @author chendong
 */
public class ModalModule extends BaseModule {

    @JSMethod
    public void toast(String msg){
        ToastUtils.show(msg);
    }

    @JSMethod
    public void toastLong(String msg){
        ToastUtils.showLong(msg);
    }
}

File: src/main/java/com/march/wxcube/module/ModalModule.kt
package com.march.wxcube.module

import com.march.common.utils.ToastUtils
import com.taobao.weex.annotation.JSMethod
import com.taobao.weex.ui.module.WXModalUIModule

/**
 * CreateAt : 2018/4/18
 * Describe : 模态，主要是 toast/alert/confirm
 *
 * @author chendong
 */
class ModalModule : WXModalUIModule() {

    /**
     * toast
     */
    @JSMethod(uiThread = true)
    fun toast(msg: String) {
        ToastUtils.show(msg)
    }

    /**
     * toast long
     */
    @JSMethod(uiThread = true)
    fun toastLong(msg: String) {
        ToastUtils.showLong(msg)
    }
}